browser/main.js

```diff
@@ -1,11 +1,10 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
+var ContestForm = require('./components/contest_form.js');
 
 document.addEventListener('DOMContentLoaded', function() {
   ReactDOM.render(
-    <h1>Hello, world!</h1>,
+    <ContestForm url='http://127.0.0.1:8080/' />,
```

Comment:
This must be taken from a global state, for example window.config.backend_address  or something like that.

In this way you can make changes easily.